corehq/messaging/management/commands/export_conditional_alerts.py

```diff
@@ -0,0 +1,262 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from corehq.apps.data_interfaces.models import (
+    AutomaticUpdateRule,
+    MatchPropertyDefinition,
+    CreateScheduleInstanceActionDefinition,
+)
+from corehq.messaging.scheduling.models import (
+    Schedule,
+    AlertSchedule,
+    TimedSchedule,
+    TimedEvent,
+    SMSContent,
+)
+from corehq.messaging.scheduling.scheduling_partitioned.models import CaseScheduleInstanceMixin
+from django.core.management.base import BaseCommand, CommandError
+from io import open as io_open
+import copy
+import json
+import jsonobject
+import six
+
+
+SIMPLE_SMS_DAILY_SCHEDULE_WITH_TIME = 1
+SIMPLE_SMS_ALERT_SCHEDULE = 2
+
+
+class MatchPropertyCriterion(jsonobject.JsonObject):
+    property_name = jsonobject.StringProperty()
+    property_value = jsonobject.StringProperty()
+    match_type = jsonobject.StringProperty()
+
+
+class SimpleSchedulingRule(jsonobject.JsonObject):
+    name = jsonobject.StringProperty()
+    case_type = jsonobject.StringProperty()
+    criteria = jsonobject.ListProperty(MatchPropertyCriterion)
+    recipients = jsonobject.ListProperty(jsonobject.ListProperty(jsonobject.StringProperty(required=False)))
+    reset_case_property_name = jsonobject.StringProperty()
+    start_date_case_property = jsonobject.StringProperty()
+    specific_start_date = jsonobject.DateProperty()
+    scheduler_module_info = jsonobject.ObjectProperty(CreateScheduleInstanceActionDefinition.SchedulerModuleInfo)
+
+
+class ExtraSchedulingOptions(jsonobject.JsonObject):
+    active = jsonobject.BooleanProperty()
+    include_descendant_locations = jsonobject.BooleanProperty()
+    default_language_code = jsonobject.StringProperty()
+    custom_metadata = jsonobject.DictProperty(six.text_type)
+    use_utc_as_default_timezone = jsonobject.BooleanProperty()
+    user_data_filter = jsonobject.DictProperty(jsonobject.ListProperty(six.text_type))
+    stop_date_case_property_name = jsonobject.StringProperty()
+
+
+class SimpleSMSDailyScheduleWithTime(jsonobject.JsonObject):
+    schedule_type = SIMPLE_SMS_DAILY_SCHEDULE_WITH_TIME
+    time = jsonobject.TimeProperty()
+    message = jsonobject.DictProperty(six.text_type)
+    total_iterations = jsonobject.IntegerProperty()
+    start_offset = jsonobject.IntegerProperty()
+    start_day_of_week = jsonobject.IntegerProperty()
+    extra_options = jsonobject.ObjectProperty(ExtraSchedulingOptions)
+    repeat_every = jsonobject.IntegerProperty()
+
+
+class SimpleSMSAlertSchedule(jsonobject.JsonObject):
+    schedule_type = SIMPLE_SMS_ALERT_SCHEDULE
+    message = jsonobject.DictProperty(six.text_type)
+    extra_options = jsonobject.ObjectProperty(ExtraSchedulingOptions)
+
+
+def open_for_json_write(path):
+    # json.dumps returns bytes in python2, but unicode in python3
+    if six.PY2:
+        return open(path, 'wb')
```

Comment:
I thought the point of the `six` package was to provide functions that work for both py2 and py3, is there something that could be used here @nickpell?

Reply:
Writing json to a file is a complicated special case that Daniel summarized here: https://github.com/dimagi/langcodes/pull/19#discussion_r189239788

Reply:
thats helpful, thanks!